tools/cloud-build/daily-tests/builds/packer-v5-legacy.yaml

```diff
@@ -27,7 +27,7 @@ timeout: 14400s  # 4hr
 steps:
 # test image creation by provisioning a new VPC and using Packer to build an
 # image in it
-- id: packer
+- id: packer-v5
```

Comment:
I don't think it is necessary to change any step ids. We use this in some test infra to track failures associated with certain tests.

Reply:
Updated back to previous one.